tests/test_ApertureFlux.py

```diff
@@ -70,7 +70,7 @@ def testNaive(self):
                                                          position)
                 area = self.computeNaiveArea(position, radius)
                 # test that this isn't the same as the sinc flux
-                self.assertNotClose(
+                self.assertFloatsNotEqual(
```

Comment:
Wasn't immediately clear why you did this — I had to go and look up the definition of `assertNotClose` to find out that it's deprecated. Maybe worth noting that in the commit message?